lib/resolv.rb

```diff
@@ -2310,6 +2315,57 @@ def self.decode_rdata(msg) # :nodoc:
             return self.new(priority, weight, port, target)
           end
         end
+
+        ##
+        # CAA resource record defined in RFC 6844
+        #
+        # These records identify certificate authority allowed to issue
+        # certificates for the given domain.
+
+        class CAA < Resource
+          TypeValue = 257
+          ClassValue = IN::ClassValue
+          ClassHash[[TypeValue, ClassValue]] = self # :nodoc:
+
+          ##
+          # Flag:
+          #   Bit 0 : 0 = not critical, 1 = critical
+
+          attr_reader :flag
+
+          ##
+          # Tag: issue, issuewild, iodef…
+
+          attr_reader :tag
+
+          ##
+          # Value of the tag.
+
+          attr_reader :value
+
+          ##
+          # Creates a new CAA for +flag+, +tag+ and +value+.
+
+          def initialize(flag, tag, value)
+            @flag, @tag, @value = flag, tag, value
```

Comment:
Elsewhere in this file assignment to instance variables is done using one line per assignment. Perhaps you want to align with the existing code-style?

Reply:
Fixed :)